homeassistant/components/sisyphus/__init__.py

```diff
@@ -29,19 +29,15 @@
     {DOMAIN: vol.Any(AUTODETECT_SCHEMA, TABLES_SCHEMA)}, extra=vol.ALLOW_EXTRA
 )
 
+# Silence these loggers by default. Their INFO level is super chatty and we
+# only need error-level logging from the integration itself by default.
+logging.getLogger("socketio.client").setLevel(logging.CRITICAL + 1)
```

Comment:
The downside of this is that this integration is now setting it for any integration that might use SocketIO. Can you target a more specific logger tied to the SocketIO sisyphus instance?

Reply:
Considering the logging guidelines for integrations, that might not be that big of a downside. Here's about a minute's worth of logs from this thing:
```
2022-02-18 07:48:30 INFO (MainThread) [socketio.client] Received event "set" [/]
2022-02-18 07:48:40 INFO (MainThread) [socketio.client] Received event "set" [/]
2022-02-18 07:48:40 INFO (MainThread) [socketio.client] Received event "set" [/]
2022-02-18 07:48:50 INFO (MainThread) [socketio.client] Received event "set" [/]
2022-02-18 07:49:00 INFO (MainThread) [socketio.client] Received event "set" [/]
2022-02-18 07:49:10 INFO (MainThread) [socketio.client] Received event "set" [/]
2022-02-18 07:49:10 INFO (MainThread) [socketio.client] Received event "set" [/]
2022-02-18 07:49:20 INFO (MainThread) [socketio.client] Received event "set" [/]
2022-02-18 07:49:23 INFO (MainThread) [socketio.client] Received event "set" [/]
2022-02-18 07:49:30 INFO (MainThread) [socketio.client] Received event "set" [/]
```

It's looks like it's logging every message it gets at INFO level. :(

> Can you target a more specific logger tied to the SocketIO sisyphus instance?

Is there some `logger` magic you have in mind for that, or would it be something that SocketIO would have to expose in its API?